docs(changelog): changelog 2022-12-15 (#1098)
scaleway/docs-content: changelog/observability/december2022/2022-12-15-grafana-v2.mdx

---
title: Grafana as a Service v2 is now in production!
status: added
author:
    fullname: 'Join the #observability-beta channel on Slack.'
    url: 'https://slack.scaleway.com/'
date: 2022-12-15
category: observability
product: cockpit
---

> The team worked hard to deliver a new version of our Grafana as a Service platform, improving performance by a 5x factor. You should not feel the “cold start” time anymore when performing an infrequent query to your grafana dashboards into your Cockpit. The feature is accessible for all users in [public beta](https://www.scaleway.com/en/betas/#observability).

Author: ```suggestion
We launched a new version of our Grafana as a Service platform, improving performance by a 5x factor. You should not feel the “cold start” time anymore when performing an infrequent query to your Grafana dashboards into your Cockpit. The feature is accessible to all users in [public beta](https://www.scaleway.com/en/betas/#observability).
```
